qiskit/providers/aer/pulse_new/models/quantum_models.py

```diff
@@ -0,0 +1,245 @@
+
+        self.noise_signals = noise_signals
+
+    @property
+    def vectorized_lindblad_generator(self):
+        """Get the :class:`OperatorModel` representing the vectorized Lindblad
+        equation (described in the class doc string), in column-stacking
+        convention.
+
+        In column stacking convention, the map :math:`X \mapsto -i[H(t), X]`
+        is mapped to :math:`-i(id \otimes H(t) - H(t)^T \otimes id)`,
+        and a dissipation term
+        :math:`X \mapsto LXL^\dagger - \frac{1}{2}\{L^\dagger L,X\}` is
+        given as
+        :math:`\overline{L} \otimes L - \frac{1}{2}(id \otimes L^\daggerL + \overline{L^\dagger L} \otimes id)`.
+
+        This function turns every operator in the :class:`HamiltonianModel`
+        into the vectorized version of :math:`-i[H_j, \cdot]`, every
+        operator in `self.noise_operators` into the vectorized dissipator
+        above, and concatenates these lists of operators, as well as the
+        signals corresponding to both, to form a new :class:`OperatorModel`.
+
+        Returns:
+            OperatorModel: corresponding to vectorized Lindblad equation.
+        """
+
+        # combine operators
+        vec_ham_ops = -1j * vec_commutator(to_array(self.hamiltonian._operators))
+
+        full_operators = None
+        if self.noise_operators is not None:
+            vec_diss_ops = vec_dissipator(to_array(self.noise_operators))
+            full_operators = np.append(vec_ham_ops, vec_diss_ops, axis=0)
+        else:
+            full_operators = vec_ham_ops
+
+        # combine signals
+        # it will take some thought to make this nice but for now I'll just
+        # put it together quickly
+        ham_sigs = self.hamiltonian.signals
+
+        full_signals = None
+        if self.noise_operators is not None:
+            noise_sigs = None
+            if isinstance(self.noise_signals, VectorSignal):
+                noise_sigs = self.noise_signals
+            elif isinstance(self.noise_signals, list):
+                noise_sigs = VectorSignal.from_signal_list(self.noise_signals)
+
+            full_envelope = lambda t: np.append(ham_sigs.envelope(t),
+                                                noise_sigs.envelope(t))
+            full_carrier_freqs = np.append(ham_sigs.carrier_freqs,
+                                          noise_sigs.carrier_freqs)
+
+            full_drift_array = np.append(ham_sigs.drift_array,
+                                         noise_sigs.drift_array)
+
+            full_signals = VectorSignal(envelope=full_envelope,
+                                        carrier_freqs=full_carrier_freqs,
+                                        drift_array=full_drift_array)
+        else:
+            full_signals = ham_sigs
+
+
+        return OperatorModel(operators=full_operators,
+                             signals=full_signals)
+
+
+def to_array(op: Union[Operator, np.array, List[Operator], List[np.array]]):
```

Comment:
You could probably just replace this function with `Operator(op).data`

Reply:
See comment below.